ambry-router/src/main/java/com/github/ambry/router/SimpleOperationTracker.java

```diff
@@ -248,6 +248,18 @@ class SimpleOperationTracker implements OperationTracker {
         // Here set the success target to the number of eligible replicas.
         replicaSuccessTarget = eligibleReplicas.size();
         break;
+      case ReplicateBlobOperation:
+        // Replicate one blob. Use the same "success target" and "request parallelism" config settings as PutOperation.
+        // Only difference is for ReplicateBlobOperation, crossColoEnabled is true.
+        // ON_DEMAND_REPLICATION_TODO: we may consider add its own configure setting if needed.
+        eligibleReplicas = getEligibleReplicas(datacenterName, EnumSet.of(ReplicaState.STANDBY, ReplicaState.LEADER));
+        replicaSuccessTarget =
```

Comment:
the sucessTarget is too large for ReplicateBlob. If we only have 1 source datanode and 6 remote colo nodes that are missing data and eligiable, then the successTarget is 5 from the code. We don't need that many replicas to have this blob.

Reply:
Intended to use successTarget as 2.